OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -99,6 +100,14 @@ public void onPageScrolled(int position, float positionOffset, int positionOffse
 
             @Override
             public void onPageSelected(int position) {
+                // Disable ViewPager2 scrolling to enable horizontal scrolling to for the WebView (Specific HTML Components).
+                if (pagerAdapter.getUnit(position).getType() == BlockType.DRAG_AND_DROP_V2) {
```

Comment:
As per discussion on ticket, we are not showing any msg right now so we can write it as 
`pager2.setUserInputEnabled(pagerAdapter.getUnit(position).getType() != BlockType.DRAG_AND_DROP_V2);`

Reply:
updated.